docs/core/linux-prerequisites.md

```diff
@@ -71,15 +85,18 @@ CentOS distributions require the following libraries installed:
 * zlib
 * libicu-devel
 
-## Installing .NET Core Prerequisites With the Native Installers
+For more information about the dependencies, see [Self contained Linux applications](https://github.com/dotnet/core/blob/master/Documentation/self-contained-linux-apps.md).
```

Comment:
It should be Self-contained. I made a PR to that topic to fix that and other things.